res/controllers/Keyboard-Default.kbd.xml

```diff
@@ -0,0 +1,382 @@
+<?xml version="1.0" ?>
+<MixxxKeyboardPreset mixxxVersion="2.0.1+" schemaVersion="1">
+  <info>
+    <name>Default Mapping</name>
+    <author>Mixxx</author>
+    <description>Default mapping for Mixxx.</description>
+  </info>
+  <controller>
+    <group name="[AutoDJ]">
+      <control action="enabled">
+        <keyseq final="1">Shift+F12</keyseq>
+      </control>
+      <control action="fade_now">
+        <keyseq final="1">Shift+F11</keyseq>
+      </control>
+      <control action="shuffle_playlist">
+        <keyseq final="1">Shift+F9</keyseq>
+      </control>
+      <control action="skip_next">
+        <keyseq final="1">Shift+F10</keyseq>
+      </control>
+    </group>
+    <group name="[Channel1]">
+      <control action="LoadSelectedTrack">
+        <keyseq final="1">Shift+Left</keyseq>
+      </control>
+      <control action="back">
+        <keyseq lang="en_US" scancode="30">a</keyseq>
```

Comment:
It looks like we have redundant informations in this line, a double binding.
You bind to "en_US-a" and to scancode "30" luckily  it is the same here.  
How about supporting two elements keyseq and scancode: 
```
 <keyseq>a</keyseq>  // binds to the "a" labeled key on all not explicit listed layouts (old final)
 <keyseq lang="en_US">a</keyseq>  // binds to the "a" labled key on a "en_US" layout, 
 <scancode>30</scancode>  // binds to scancode "30" on all not explicit listed layouts
 <scancode>
   <keyseq lang="en_US">a</keyseq>
 </scancode>  // binds to scancode of the "en_US" "a" on all not explicit listed layouts. 
```